jekyll/content/terms/generative-adversarial-network-gan.md

```diff
@@ -5,3 +5,9 @@ references:
  - "[Generative adversarial networks](https://arxiv.org/abs/1406.2661)"
  - "[NIPS 2016 tutorial: Generative Adversarial Networks](https://arxiv.org/abs/1701.00160)"
 ---
+Generative adversarial networks were proposed by Ian Goodfellow in 2014. A GAN consists of a model with two neural networks, G (generative model) and D (discriminative model), which are competing in a minimax two-player game.
+
+The neural network G is modeled to capture the data distribution, and it's purpose is to generate new content. The network D is the discriminator, which purpose is to estimate the probability that a sample came from the training data rather then generated by G.
+The training of the G network ultimately is to maximize the probability of D making a mistake (minimax strategy). The mutual training of the GAN creates a form of competition where D is getting better at the prediction (distinguishing between real and generated data), and G is learning to become less predictable.
+
+GANs are difficult to train due to the fact that the training of G and D must be well synchronized. If D or G becomes too good compared to the other, a GAN won't converge (Helvetica scenario). GANs don't need to use Markov chains and no inference on the data is needed in the learning sessions.
```

Comment:
I think it is worth mentioning mode collapse / the Helvetica scenario, but I wouldn't say that "GANs are difficult to train" because that statement will become out-of-date as soon as someone publishes a paper that makes GANs easy to train.

And then we would be wrong! :)